The doc subpkg of speex fails the doc test

The documentation subpackage of speex fails testing with the following error:

```
2025/02/04 22:39:56 WARN + grep -q ^Name: usr/lib/pkgconfig/speex.pc
2025/02/04 22:39:56 WARN + basename usr/lib/pkgconfig/speex.pc .pc
2025/02/04 22:39:56 WARN + lib_name=speex
2025/02/04 22:39:56 WARN + echo usr/lib/pkgconfig/speex.pc
2025/02/04 22:39:56 WARN + grep -q '^usr/lib/pkgconfig/speex.pc$\|^usr/share/pkgconfig/speex.pc$'
2025/02/04 22:39:56 WARN + pkgconf --exists speex
2025/02/04 22:39:56 WARN + grep -q ^Version: usr/lib/pkgconfig/speex.pc
2025/02/04 22:39:56 WARN + pkgconf --modversion speex
2025/02/04 22:39:56 INFO 1.2.1
2025/02/04 22:39:56 WARN + grep -q ^Libs: usr/lib/pkgconfig/speex.pc
2025/02/04 22:39:56 WARN + pkgconf --libs speex
2025/02/04 22:39:56 INFO -lspeex
2025/02/04 22:39:56 WARN + grep -q ^Cflags: usr/lib/pkgconfig/speex.pc
2025/02/04 22:39:56 WARN + pkgconf --cflags speex
2025/02/04 22:39:56 INFO
2025/02/04 22:39:56 WARN + '[' true '=' false ]
2025/02/04 22:39:56 WARN + exit 0
2025/02/04 22:39:56 INFO running test pipeline for subpackage speex-doc
2025/02/04 22:39:56 INFO building test workspace in: '/home/user/tmp/melange-guest-774568214-speex-doc' with apko
2025/02/04 22:39:56 [DEBUG] GET https://packages.wolfi.dev/os/apk-configuration
2025/02/04 22:39:56 INFO setting apk repositories: [/home/user/src/wolfi-os/packages https://packages.wolfi.dev/os]
2025/02/04 22:39:56 INFO image configuration:
2025/02/04 22:39:56 INFO   contents:
2025/02/04 22:39:56 INFO     build repositories: []
2025/02/04 22:39:56 INFO     runtime repositories: []
2025/02/04 22:39:56 INFO     keyring:      []
2025/02/04 22:39:56 INFO     packages:     [speex-doc apk-tools grep man-db texinfo]
2025/02/04 22:39:56 INFO installing ca-certificates-bundle (20241121-r1)
2025/02/04 22:39:56 INFO installing wolfi-baselayout (20230201-r16)
2025/02/04 22:39:56 INFO installing glibc (2.40-r8)
2025/02/04 22:39:56 INFO installing ld-linux (2.40-r8)
2025/02/04 22:39:56 INFO installing libgcc (14.2.0-r8)
2025/02/04 22:39:56 INFO installing glibc-locale-posix (2.40-r8)
2025/02/04 22:39:56 INFO installing zlib (1.3.1-r5)
2025/02/04 22:39:56 INFO installing libcrypto3 (3.4.0-r6)
2025/02/04 22:39:56 INFO installing libssl3 (3.4.0-r6)
2025/02/04 22:39:56 INFO installing apk-tools (2.14.4-r2)
2025/02/04 22:39:56 INFO installing libpcre2-8-0 (10.44-r2)
2025/02/04 22:39:56 INFO installing grep (3.11-r2)
2025/02/04 22:39:56 INFO installing libpipeline (1.5.8-r0)
2025/02/04 22:39:56 INFO installing libbz2-1 (1.0.8-r10)
2025/02/04 22:39:56 INFO installing libxcrypt (4.4.38-r0)
2025/02/04 22:39:56 INFO installing libcrypt1 (2.40-r8)
2025/02/04 22:39:56 INFO installing perl (5.40.1-r0)
2025/02/04 22:39:56 INFO installing libstdc++ (14.2.0-r8)
2025/02/04 22:39:56 INFO installing groff (1.23.0-r3)
2025/02/04 22:39:56 INFO installing libseccomp (2.6.0-r0)
2025/02/04 22:39:56 INFO installing gdbm (1.24-r2)
2025/02/04 22:39:56 INFO installing man-db (2.13.0-r1)
2025/02/04 22:39:56 INFO installing speex-doc (1.2.1-r1)
2025/02/04 22:39:56 INFO installing ncurses-terminfo-base (6.5_p20241228-r0)
2025/02/04 22:39:56 INFO installing ncurses (6.5_p20241228-r0)
2025/02/04 22:39:56 INFO installing texinfo (7.2-r0)
2025/02/04 22:39:57 INFO installing wolfi-keys (1-r8)
2025/02/04 22:39:57 INFO installing busybox (1.37.0-r0)
2025/02/04 22:39:57 INFO installing wolfi-base (1-r6)
2025/02/04 22:39:57 INFO OCI layer digest: sha256:ad6cd943d01f0e727fbc7af923d20a15cb5abe87d31c1fbae02533d6fe2a4e5b
2025/02/04 22:39:57 INFO OCI layer diffID: sha256:9bfdc601853f77c5e8f91fb1c756e183c0d6897fcf6bc072db75a2c1c56b76d8
2025/02/04 22:39:57 INFO saving OCI image locally: apko.local/cache:3341e31ddf14dc6fa1074f7d0e085f669dcc014724b97fe295c903f3af518327
2025/02/04 22:39:58 WARN skipping local domain tagging apko.local/cache:3341e31ddf14dc6fa1074f7d0e085f669dcc014724b97fe295c903f3af518327 as index.docker.io/library/melange:latest
2025/02/04 22:39:58 INFO running step "test/docs"
2025/02/04 22:39:58 WARN + '[' -d /home/build ]
2025/02/04 22:39:58 WARN + cd /home/build
2025/02/04 22:39:58 WARN + exit 0
2025/02/04 22:39:58 INFO running step "docs readability check"
2025/02/04 22:39:59 WARN + '[' -d /home/build ]
2025/02/04 22:39:59 WARN + cd /home/build
2025/02/04 22:39:59 WARN + basename /home/build/melange-out/speex-doc
2025/02/04 22:39:59 WARN + doc_pkg=speex-doc
2025/02/04 22:39:59 WARN + apk info -L speex-doc
2025/02/04 22:39:59 WARN + wc -l
2025/02/04 22:39:59 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:39:59 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:39:59 WARN + '[' 3 -le 3 ]
2025/02/04 22:39:59 WARN + echo See:
2025/02/04 22:39:59 WARN + apk info -L speex-doc
2025/02/04 22:39:59 INFO See:
2025/02/04 22:39:59 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory
2025/02/04 22:39:59 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory
2025/02/04 22:39:59 INFO speex-doc-1.2.1-r1 contains:
2025/02/04 22:39:59 INFO var/lib/db/sbom/speex-doc-1.2.1-r1.spdx.json
2025/02/04 22:39:59 INFO
2025/02/04 22:39:59 WARN + echo 'This package [speex-doc] is completely empty (i.e. installs no files).'
2025/02/04 22:39:59 WARN + echo 'Please check the package build for proper docs installation, and either:'
2025/02/04 22:39:59 WARN + echo '  (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or'
2025/02/04 22:39:59 WARN + echo '  (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or'
2025/02/04 22:39:59 WARN + echo '  (c) remove the docs subpackage entirely'
2025/02/04 22:39:59 WARN + exit 1
2025/02/04 22:39:59 INFO This package [speex-doc] is completely empty (i.e. installs no files).
2025/02/04 22:39:59 INFO Please check the package build for proper docs installation, and either:
2025/02/04 22:39:59 INFO   (a) fix the docs subpackage build to actually include documentation (check the split/manpages and split/infodir pipelines), or
2025/02/04 22:39:59 INFO   (b) remove the test/docs pipeline (if for some reason we want an empty docs package), or
2025/02/04 22:39:59 INFO   (c) remove the docs subpackage entirely
2025/02/04 22:39:59 INFO pod f2a9cdc0c9584cd3b29ab1d249b12af542199d643ff7df99ecba5b6b2d0c53a4 terminated
2025/02/04 22:39:59 INFO pod ca7a1435521253d484aedbb0963d1e00f61fedf15cdf9e3a92285e9e42006444 terminated
2025/02/04 22:39:59 INFO ERROR: failed to test package. the test environment has been preserved:
2025/02/04 22:39:59 INFO   workspace dir: /home/user/tmp/melange-workspace-3765892005
2025/02/04 22:39:59 INFO   guest dir: /home/user/tmp/melange-guest-774568214
2025/02/04 22:39:59 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:121: test/speex] Error 1
```

--- a/speex.yaml
+++ b/speex.yaml
@@ -2,7 +2,7 @@
 package:
   name: speex
   version: 1.2.1
-  epoch: 1
+  epoch: 2
   description: an audio compression format designed for speech
   copyright:
     - license: BSD-3-Clause
@@ -55,7 +55,13 @@ subpackages:
   - name: speex-doc
     pipeline:
       - uses: split/manpages
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/share
+          mv "${{targets.destdir}}"/usr/share/doc "${{targets.subpkgdir}}"/usr/share
     description: speex manpages
+    test:
+      pipeline:
+        - uses: test/docs
 
 update:
   enabled: true

